src/main/java/org/prebid/server/bidder/OpenrtbBidder.java

```diff
@@ -231,7 +229,7 @@ protected void modifyRequest(BidRequest bidRequest, BidRequest.BidRequestBuilder
                 .filter(Objects::nonNull)
                 .flatMap(Collection::stream)
                 .map(bid -> BidderBid.of(bid, getBidType(bid.getImpid(), bidRequest.getImp()),
-                        getBidCurrency(bidRequest.getCur(), bidResponse.getCur())))
+                        StringUtils.stripToNull(bidResponse.getCur())))
```

Comment:
Sorry, but i have a crazy thought about `stripToNull` action performing.

It is kind frail to add this in each bidder, may be for simplicity it is reasonable to leave it as is like: `bidResponse.getCur()` without any processing. Then, in `ExchangeService` (before currency conversion service call) we can add this `stripToNull` action.
So, just in one place - before actual using the currency value.

What do you think?

Reply:
Yah, good question about performance. I agree with you. I'll try to change the logic